cinema/management/commands/wait_for_db.py

```
@@ -0,0 +1,20 @@
+import time as t
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    help_text = "Wait for the database to become available"
+
+    def handle(self, *args, **kwargs):
+        self.stdout.write("⏳ Waiting for database...")
+        while True:
+            try:
+                db_conn = connections["default"]
+                db_conn.cursor()
+                break
+            except OperationalError:
```

Currently only `OperationalError` is caught. In some database backends the low-level exception may differ or more context could be helpful. Either catch and log the exception instance (`except OperationalError as e:`) or consider broadening handling to expected DB connection exceptions while avoiding swallowing unrelated errors. Ensure you still re-raise unexpected exceptions if appropriate.